schemas/github-pull-request-message.yml

```diff
@@ -0,0 +1,39 @@
+id: http://schemas.taskcluster.net/github/v1/github-pull-request-message.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "GitHub Pull Request Message"
+description: |
+  Message reporting that a GitHub pull request has occurred
+type:         object
+properties:
+  version:    {$const: message-version}
+  organization:
+    description: |
+      The GitHub `organization` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  repository:
+    description: |
+      The GitHub `repository` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  action:
+    description: |
+      The GitHub `action` which triggered an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    22
+    pattern:      {$const: identifier-pattern}
```

Comment:
If this is a fixed set them let's specify it as an `enum` instead... it's much easier to consume the messages when we know all possible values. See: 
http://spacetelescope.github.io/understanding-json-schema/reference/generic.html#enumerated-values

And then please put in the description that you may add additional entries in the future. Then people can consume messages in a reliable manner ignoring future additions to the enum.

Note, if github has docs for these properties you should like to that... and note, description is markdown, so you can include links :)